src/chi/components/tables/tables.scss

```diff
@@ -134,6 +134,60 @@ th {
     }
   }
 
+  &.-portal {
+    thead {
+      tr {
+        th {
+          background-color: set-color(grey, 25);
+          border-bottom: 0;
+          font-weight: 700;
+          padding-bottom: 0.75rem;
+          padding-top: 0.75rem;
+
+          &:nth-child(2) {
+            text-align: right;
+          }
+        }
+      }
+    }
+
+    tbody {
+      tr {
+        &:nth-child(odd) {
+          background-color: set-color(grey, 20);
+        }
+
+        &.-active {
+          & > td {
+            background-color: rgb(206, 218, 241);
+
+            &:first-child {
+              border-left: 0.25rem solid $primary-color;
+              padding-left: 0.75rem;
+            }
+          }
+        }
+
+        td {
+          border-bottom: 0;
+
+          a {
+            color: set-color(navy, 70);
+            font-weight: 700;
+
+            &:visited {
+              color: set-color(navy, 70);
+            }
+          }
+
+          &:nth-child(2) {
```

Comment:
@janibolkvadze please remove this selector as it can be achieved with our alignment classes for tables.